docs/2_4_common_schema.adoc

```diff
@@ -1081,8 +1081,7 @@ The default value of this attribute is `false`.
 |
 [[previous,`previous`]]
 If present, this variable is a <<ClockedState>> and this attribute is a value reference to the variable with the previous value.
-Only clocked variables (i.e. variables with the attribute <<clocks>>) may have the <<previous>> attribute.
-Only variables with <<variability,`variability == discrete`>> may have a previous value.
+The following constraints apply: only clocked variables (i.e. variables with the attribute <<clocks>>) may have the <<previous>> attribute; only variables with <<variability,`variability == discrete`>> may have a previous value; and the variable referenced by the previous attribute must have the same type.
```

Comment:
@andreas-junghanns don't you think "same type" is too strong? The reader may think that a ClockedState can only refer to a previous ClockedState (and thus having an infinite chain or a self reference at some point). We want to allow that a ClockedState refers to another Discrete variable (thus the "compatible type" expression)

Reply:
I see. We should be clear: It must have the same numeric type. Is "numeric" sufficient and clear?

Reply:
I think it is.

Reply:
fixed - merge?